src/components/navBar/index.tsx

```diff
@@ -1,17 +1,106 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { USER_SELF } from '../constants/url'
+import navLinks from './navlink'
 
 const RDSLogo = '/RDSLogo.png';
+const GitHubLogo = '/gitHubLogo.svg'
```

Comment:
We need to use a GitHub logo with white fill as opposed to the current one. 
Can we please change it?
https://crypto.realdevsquad.com/assets/github.png

![image](https://user-images.githubusercontent.com/46373689/163728638-3ae319da-d940-4075-b997-e6502c45108e.png)

Reply:
Fixed